stab.py

```diff
@@ -1,106 +1,76 @@
-import time, random, sys
-answer = "gyhgftyhg"
-
-def type(str):
-  for letter in str:
-    sys.stdout.write(letter)
-    sys.stdout.flush()
-    time.sleep(0.05)
-  sys.stdout.write("\n")
-  sys.stdout.flush()
-
-def comptype(str):
-  for letter in str:
-    sys.stdout.write(letter)
-    sys.stdout.flush()
-    time.sleep(0.666)
-  sys.stdout.write("\n")
-  sys.stdout.flush()
-
-def slowtype(str):
-  for letter in str:
-    sys.stdout.write(letter)
-    sys.stdout.flush()
-    time.sleep(0.33)
-  sys.stdout.write("\n")
-  sys.stdout.flush()
-
-def laugh(str):
-  for letter in str:
-    sys.stdout.write(letter)
-    sys.stdout.flush()
-    time.sleep(0)
-  sys.stdout.flush()
```

Comment:
`laugh` wasn't printing new lines, maybe `dur_type` could handle this?

Reply:
I just checked, the only call to laugh (from stab.py) can and should have a new line at the end